rudder-web/src/main/scala/com/normation/rudder/web/snippet/BootstrapMenu.scala

```diff
@@ -77,11 +80,12 @@ class BootstrapMenu {
         }
       }
     }
-
-    <ul class="nav navbar-nav">
+    <ul class="sidebar-menu">
```

Comment:
All the html should be in common-layout (in fact the part with the version should be in common-layout.html)